### PR TITLE
contributing and improvements

### PR DESCRIPTION
**Pull request by bglusman**
@devmanhinton edits welcome.

### CODE REVIEW

**Comment by devmanhinton** [comment]
@bglusman Perhaps we should be a bit less cheery :) ? 

I am guessing that a lot of programmers are introverted types and often introverts do not react well to super friendly text with lots of "!!!".

Also the standard seems to be a dry tone:

http://emberjs.com/guides/contributing/adding-new-features/
http://docs.angularjs.org/misc/contribute

**Comment by sethbannon** [comment]
There is a typo in the lsat paragraph. "Right now EndDash only support browser..." should be "Right now EndDash only supports browser..."

File: _includes/README.md [hunk 1]
```diff
@@ -629,6 +629,44 @@ update the DOM on model changes.
 EndDash used without Backbone style getting & setting will still interpolate
 a bound object's attributes into the DOM but will not update the DOM on model changes.
 
+Contributing and Future Improvements
+====================================
+
+There are a number of future improvements planned for EndDash and we'd love for you
```
Comment (yaymukund) [inline _includes/README.md:635]: I feel like Ray wrote this paragraph lol

Comment (tobowers) [inline _includes/README.md:635]: haha.  You get _one_ exclamation mark per _section_

Comment (bglusman) [inline _includes/README.md:635]: I was in a hurry, cut me some slack :-)

**Comment by bglusman** [comment]
@devmanhinton addressed your and seth's comments I think, merge?

**Pull request merged**